### ISSUE TITLE
Can't edit existing connection entries

### ISSUE THREAD

**Issue by masscream** [asbru-cm/asbru-cm#612]
**Describe the bug**
Can't call the connection parameters window again I usually get anytime I create new connection.

**To Reproduce**
1. Right click on connection on the left
2. Click on 'Edit connection or ALT+E'
3. Nothing pops up

**Expected behavior**
Connection parameters window should appear

**Error code, when asbru run through terminal**
```
*** unhandled exception in callback:
***   undefined value for mandatory argument 'text' encountered at /opt/asbru/lib/PACEdit.pm line 703.
***  ignoring at /usr/lib/x86_64-linux-gnu/perl5/5.28/Glib/Object/Introspection.pm line 67.
```

**Environment (please complete the following information):**
 - OS: Debian 10 Buster, Xfce 4.12
 - Ásbrú Connection Manager 6.1.1

**Additional context**
I get the window normally when I hit Create new connection button. But not when I try to modify existing ones. It used to work a few days / weeks back, not sure what I did to stop it working (could be updates). I also thought it was an "another monitor" issue which may happen when I connect new monitor to the laptop and from time to time windows remember to load up on the monitor which is not even connected. (I saw this with one specific app on windows, but that's another story). But this time the window does not pop up at all.

**Comment by hanspr** [comment]
Could you update to the must recent version, we are on 6.2

If it is not available in your package manager, follow installation instructions below.

https://docs.asbru-cm.net/General/Installation/

**Comment by gfrenoy** [comment]
This has indeed already been fixed ; see #572.  You should at least upgrade to 6.1.3.
